stingray/pulse/search.py

```diff
@@ -373,14 +400,21 @@ def phaseogram(times, f, nph=128, nt=32, ph0=0, mjdref=None, fdot=0, fddot=0,
     allphases = np.concatenate([phases, phases + 1]).astype('float64')
     allts = \
         np.concatenate([times, times]).astype('float64')
+    if weights is not None and isinstance(weights, collections.Iterable):
+        if len(weights) != len(times):
+            raise ValueError('The length of weights must match the length of '
+                             'times')
+        weights = \
+            np.concatenate([weights, weights]).astype('float64')
```

Comment:
I'm not sure I understand what's going on here. Why do you duplicate the weights?

Reply:
Because I duplicated the times before :). 
The phaseogram is shown duplicated, from phase 0 to 1 and repeated from 1 to 2
This is a little ugly, I admit.

Reply:
fair enough. :)